rfcs/0008-commitstatus/README.md

```diff
@@ -0,0 +1,71 @@
+
+# RFC-xxxx Create a new API kind for Commit Status
+
+**Status:** provisional
+
+**Creation date:** 2022-05-12
+
+**Last update:** 2022-05-12
+
+## Summary
+
+There should be a dedicated kind in the notification controller for sending commit status notifications to git providers.
+
+## Motivation
+
+Currently, The Alert type can reference both git providers and chat providers. However, the differences between the two providers and how notifications being sent to them should be handled has continued to diverge. 
```

Comment:
1. `Currently, the ...`
2. git -> Git
3. I would provide examples of what Git providers and/or chat providers are to provide further context.
4. The last part of this line is difficult to read ("being sent to them should be handled has continued to diverge").